fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/query/SqlConstants.java

```diff
@@ -1,5 +1,5 @@
 /*
- * (C) Copyright IBM Corp. 2019
+ * (C) Copyright IBM Corp. 2019,2021
```

Comment:
not really needed, and many of these constants are used in other spots/declared in other lcoations

Reply:
```


216 |   |   | Constant "SqlConstants.GROUP_BY" has 0 references | 0 |   | com.ibm.fhir.database.utils.query.SqlConstants.GROUP_BY(SqlConstants.java:17) | Constant | Reference
-- | -- | -- | -- | -- | -- | -- | -- | --
217 |   |   | Constant "SqlConstants.HAVING" has 0 references | 0 |   | com.ibm.fhir.database.utils.query.SqlConstants.HAVING(SqlConstants.java:18) | Constant | Reference
218 |   |   | Constant "SqlConstants.IS_NULL" has 0 references | 0 |   | com.ibm.fhir.database.utils.query.SqlConstants.IS_NULL(SqlConstants.java:19) | Constant | Reference
219 |   |   | Constant "SqlConstants.IS_NOT_NULL" is only called from tests | - |   | com.ibm.fhir.database.utils.query.SqlConstants.IS_NOT_NULL(SqlConstants.java:20) | Constant | TestOnly
220 |   |   | Constant "SqlConstants.INNER_JOIN" has 0 references | 0 |   | com.ibm.fhir.database.utils.query.SqlConstants.INNER_JOIN(SqlConstants.java:21) | Constant | Reference
221 |   |   | Constant "SqlConstants.OUTER_JOIN" has 0 references | 0 |   | com.ibm.fhir.database.utils.query.SqlConstants.OUTER_JOIN(SqlConstants.java:22) | Constant | Reference
222 |   |   | Constant "SqlConstants.MAX" has 0 references | 0 |   | com.ibm.fhir.database.utils.query.SqlConstants.MAX(SqlConstants.java:23) | Constant | Reference
223 |   |   | Constant "SqlConstants.CASE_WHEN" has 0 references | 0 |   | com.ibm.fhir.database.utils.query.SqlConstants.CASE_WHEN(SqlConstants.java:24) | Constant | Reference
224 |   |   | Constant "SqlConstants.THEN" has 0 references | 0 |   | com.ibm.fhir.database.utils.query.SqlConstants.THEN(SqlConstants.java:25) | Constant | Reference
225 |   |   | Constant "SqlConstants.END" has 0 references | 0 |   | com.ibm.fhir.database.utils.query.SqlConstants.END(SqlConstants.java:26) | Constant | Reference
226 |   |   | Constant "SqlConstants.ORDER_BY" has 0 references | 0 |   | com.ibm.fhir.database.utils.query.SqlConstants.ORDER_BY(SqlConstants.java:27)
```

Reply:
Constants refactor to be done after we purge the old builder code (after reworking whole system search)